### DISCUSSION TITLE
A option for changing mob spawning in claims

### DISCUSSION THREAD

**Discussion by Pandasquared** [GriefPrevention/GriefPrevention#832] [Q&A] [answered]
Im just wondering if there is a way i can make it so mobs will spawn in claims if there is a mob spawner a player tries to breed an animal or a spawn egg is used, If this is not currently a feature i really would love for it to be. As i cannot find an other plugin that work the way i want it like grief prevention does.  
Short:
Got a unique server im working on and i need to be able to allow mobs to spawn via spawn egg, spawner, or breeding.

**Comment by RoboMWM** [comment]
I do not understand this question, please rephrase as it seems you're contradicting yourself within the sentences.

**Reply by kalleswe** [reply]
Really?? Read it again, it was way clear to me.

**Reply by RoboMWM** [reply]
well that certainly clarifies...



...absolutely nothing, other than telling me I'm an idiot. Thanks for that, I guess. Anyways, this is a year-old discussion that was converted from an issue, and was answered by others.

**Reply by kalleswe** [reply]
You're welcome, since it's was way to clear for me. And really bringing up a subject I was searching for which can't seem to work still in GP.

**Reply by RoboMWM** [reply]
I'm glad it's way clear for you. If only you were a developer too, you could PR this option or whatever it is.

**Comment by Pandasquared** [comment]
Not really hars to understand. I want mpbs to spawn in claimed land. How hard is That to understand. 
As of rifht now players cannot breed animals or use mob spawners. As griefprevention completely blocks all spawning of mobs.
Sent from Yahoo Mail on Android 
 
  On Sun, May 31, 2020 at 12:57 PM, RoboMWM<notifications@github.com> wrote:   


I do not understand this question, please rephrase as it seems you're contradicting yourself within the sentences.

**Comment by Jikoo** [comment, marked as answer]
GP only interferes with spawning while using the creative ruleset. Use survival claim mode. You also need build trust to be able to use spawn eggs inside of claims, and there is a separate config option allowing them to be used outside of claims.

**Reply by kalleswe** [reply]
Still does not work in Survival setting at all.

**Reply by bigpresh** [reply]
> Still does not work in Survival setting at all.

Then please [raise an issue](https://github.com/TechFortress/GriefPrevention/issues/new?assignees=&labels=Unverified+Bug&template=bug-report.md&title=) filling out all the details requested by the template - in particular supplying your config, and letting us know exactly what you're trying to do (e.g. use a spawn egg inside of a claim you have build trust in) and exactly what happens, including any messages you see.

Also, please confirm if you still get the same behaviour with only Grief Prevention installed, as it's quite possible some other plugin is blocking it.

**Comment by Pandasquared** [comment]
Yeah and im asking for an option to completely override all spawning options and let me set it globallt for all claims regardless of what kins of claim it is
Sent from Yahoo Mail on Android 
 
  On Sun, May 31, 2020 at 5:04 PM, Adam<notifications@github.com> wrote:   


GP only interferes with spawning while using the creative ruleset. Use survival claim mode. You also need build trust to be able to use spawn eggs inside of claims, and there is a separate config option allowing them to be used outside of claims.

**Comment by Jikoo** [comment]
I mean, yeah, the option is to use the survival claims mode instead of creative.

If there are additional reasons you want to stick with the creative ruleset, could you explain?

**Comment by Pandasquared** [comment]
I would just like a config option to control all claims. As ive tried claiming every differwnt way i can and no matter how i claim o camt spawn mobs at all.Im setting my server up so people can raid other playera but i do not want the server to be factiona ans griefprevention ia perfect besides thia one aspect. Basically i need enemy players to be able to use creeper eggs in pther playera claims. 

Sent from Yahoo Mail on Android 
 
  On Sun, May 31, 2020 at 10:00 PM, Adam<notifications@github.com> wrote:   


I mean, yeah, the option is to use the survival claims mode instead of creative.

If there are additional reasons you want to stick with the creative ruleset, could you explain?

**Comment by Jikoo** [comment]
If you continue to have issues with mob spawning even after switching to Survival or SurvivalRequiringClaims mode, please open a bug report with all of the information the template requests.

**Reply by kalleswe** [reply]
Yeah mob spawners still doesn't work. so something must be off.
